.gitignore

```diff
@@ -1,15 +1,27 @@
 sandbox/
 example/data/*
+
+# Bay area data
 bay_area_urbansim/data/*
 bay_area_urbansim/output/*
 bay_area/data/*
 bay_area/output/*
+bay_area/tmp/*
+bay_area/pilates/*
```

Comment:
whats up with these pilates subdirs?

Reply:
This are just temporary subdirectories that I created for debugging - Should I remove them form the .gitignore? To me it was the easiest to avoid committing these folders.